### PR TITLE
fixed setting youtube host for non-https case

### PR DESCRIPTION
**Pull request by freezer278**
### Summary of proposed changes
I discovered that in some cases youtube video is not playing when it is in iFrame of non https site (e.g. localhost, etc). This pull request fixes this problem.
### Checklist
- [x] Use `develop` as the base branch
- [x] Exclude the gulp build (`/dist` changes) from the PR
- [x] Test on [supported browsers](https://github.com/sampotts/plyr#browser-support)

### CODE REVIEW

**Comment by sampotts** [comment]
Hey,

Thanks for your time. A few issues though:
- You should edit the source files not the built files (i.e. those in `/dist`)
- As the PR checklist shows, you shouldn't commit the changes to `/dist` at all
- As the PR checklist shows, you must PR to `develop` 
- You could have used `window.location.protocol` - e.g.
```javascript
const useSSL = window.location.protocol  === 'https:';
```
- It's You*T*ube. This is a common though
- This sounds more like a browser issue than an issue with Plyr itself

**Comment by sampotts** [comment]
In fact your whole function could be simplified:

```javascript
function getHost(config) {
    if (config.noCookie) {
        return 'https://www.youtube-nocookie.com';
    }

    return `${window.location.protocol}//www.youtube.com`;
}
```

Also it still boils down to the fact I'm not even sure this is required. I've _never_ ran into this before.

**Comment by freezer278** [comment]
Yes this way to write this function is much better. 
This problem is hard to reproduce because i faced it when i use plyr inside iframe that is inside another iframe, and i think it is related somehow to CORS restrictions.

**Comment by freezer278** [comment]
Made some testing in supported browsers, it works.

**Comment by sampotts** [comment]
I meant do we even need the change. Do you have an example where it fails? A codepen, jsfiddle, codesandbox or something?

File: src/js/plugins/youtube.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ function assurePlaybackState(play) {
     }
 }
 
+function getYoutubeHost(config) {
```
Comment (sampotts) [inline src/js/plugins/youtube.js]: You're already in the context of YouTube here so it can be just called `getHost`.

Comment (freezer278) [reply src/js/plugins/youtube.js]: Yes, you are right. Changed this in last commit.

**Comment by freezer278** [comment]
I am not sure that i can provide you reproduction link. This issue can be reproduced when i use plyr in scorm course that can be opened in lms system deployed locally and it is not open source project and i can`t provide you access to it. 
So i don`t know how to create such case in jsbin because they use https.
I know that this case is very rare and hard to reproduce but accepting these changes doesn't exactly hurt the project, it will only make it work in this rare case.

**Comment by sampotts** [comment]
Cheers 👍

**Pull request merged**